src/components/standalone/users_objects/DomainSets.vue

```diff
@@ -54,6 +57,7 @@ onMounted(() => {
 
 function loadData() {
   listDomainSets()
+  objects.load()
```

Comment:
I guess these changes needs to be done in `HostSets.vue` too

Reply:
Whops, forgot to add them to the commit, coming right up